### PR TITLE
Add Clerk.io types

### PR DESCRIPTION
**Pull request by madsenmm**

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/clerk.io/tsconfig.json
1. types/clerk.io/package.json
1. types/clerk.io/clerk.io-tests.ts
1. types/clerk.io/.eslintrc.json
1. types/clerk.io/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5a71c25339f7cc8dc9c14addbded83534ee0837b
</p>

File: types/clerk.io/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/export-just-namespace": "off"
```
Comment (andrewbranch) [inline types/clerk.io/.eslintrc.json:3]: Please remove this file. See 
https://github.com/microsoft/dtslint/blob/master/docs/export-just-namespace.md for how. Thanks!

**Review by andrewbranch — CHANGES_REQUESTED** [review]

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/clerk.io/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 68e36066ce81a4323237231d213b5d80e52a36cb
</p>

**Review by andrewbranch — APPROVED** [review]

**Comment by tmmgrafikr** [comment]
Ready to merge

**Pull request merged**